docs/DeveloperGuide.md

```diff
@@ -362,55 +363,239 @@ testers are expected to do more *exploratory* testing.
 
 #### **Launch and Shutdown**
 
-1. Initial launch
+#### Scenario 1  
+{: .no_toc}  
+**Context:** Initial launch
```

Comment:
Since there is an initial launch, do we need for a normal launch? (Which does not involve copying the jar file)

Reply:
Nope. I think most of the steps will be repeated. Hence over redundancy